src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -51,10 +64,11 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
             this.sessionId = Number(state.sessionId);
             this.numNewMessages = state.numNewMessages;
         });
+        this.loadFirstMessage();
     }
 
     ngAfterViewInit() {
-        this.unreadMessageIndex = this.messages.length === 0 || this.numNewMessages === 0 ? -1 : this.messages.length - this.numNewMessages;
+        this.unreadMessageIndex = this.messages.length === 1 || this.numNewMessages === 0 ? -1 : this.messages.length - this.numNewMessages;
```

Comment:
```suggestion
        this.unreadMessageIndex = this.messages.length <= 1 || this.numNewMessages === 0 ? -1 : this.messages.length - this.numNewMessages;
```
This makes it a bit more resilient in case the first message gets lost somehow.
Additionally I would expect a short comment here, why one message is also considered as read

Reply:
Basically because it's just a greeting message the chatbot will always send this if user hasn't had any chat with the chatbot before. So I don't think it's necessary to always show this as unread. 
And I'll try to add a short comment here.